src/ralph/export_to_ng/publishers.py

```diff
@@ -195,3 +197,39 @@ def sync_virtual_server_to_ralph3(sender, instance=None, created=False, **kwargs
         'parent_id': asset.id if asset else None,
         'custom_fields': _get_custom_fields(instance),
     }
+
+# TODO:
+# NetworkKind
+# NetworkEnvironment
+
+
+@ralph3_sync(Network)
+def sync_network_to_ralph3(sender, instance=None, created=False, **kwargs):
+    net = instance
+
+    def get_reserved_ips(net):
+        start = net.min_ip
+        end = net.max_ip
+        bottom, top = net.reserved, net.reserved_top_margin
+        # start + 1 , start is reserved for network address
+        for int_ip in xrange(start + 1, start + bottom + 1):
+            yield str(ipaddr.IPAddress(int_ip))
+        # end - 1 , end is reserved for broadcast address
+        for int_ip in xrange(end - 1, end - top - 1, -1):
+            yield str(ipaddr.IPAddress(int_ip))
+
+    def get_ids(manager):
+        return list(manager.all().values_list('id', flat=True))
+
+    return {
+        'address': net.address,
+        'remarks': net.remarks,
+        'vlan': net.vlan,
+        'dhcp_broadcast': net.dhcp_broadcast,
+        'gateway': net.gateway,
+        'reserved_ips': list(get_reserved_ips(net)) if net.min_ip and net.max_ip else [],  # noqa
+        'environment_id': net.environment_id,
+        'kind_id': net.kind_id,
+        'racks_ids': get_ids(net.racks),
```

Comment:
rack is device in ralph2, but dedicated model in R3. Use sth similar to https://github.com/allegro/ralph/blob/develop/src/ralph/export_to_ng/resources.py#L508